java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +375,39 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Pops one or more elements from the first non-empty list from the list of provided <code>key
+     * </code>
+     *
+     * @see <a href="https://valkey.io/commands/lmpop/">valkey.io</a> for details.
```

Comment:
Move those apiNotes to the right place

Reply:
I think you meant to include the `@apiNote` labels here.  Maybe a merge conflict issue?

Reply:
possibly, will fix, actually didnt know how they ended up with lpop